stable/aws-node-termination-handler/values.yaml

```diff
@@ -131,6 +131,11 @@ rbac:
 
 dnsPolicy: ""
 
+# Specified whether the Linux DaemonSet binds to the host network or not.
+# Enabling this gives the pod access to the loopback device, services listening
+# on localhost, and could be used to snoop on network activity of other pods on the same node
+useHostNetwork: true
```

Comment:
technically, this makes it opt-OUT :)

Reply:
Yeah true. I didn't want to break anything for anyone. Perhaps we should do a hard change here though?

Reply:
I think opt-out is the right move. Using host networking makes NTH by default work with IMDSv2. I know we've had discussions in issues about not needing it with kiam, but since eks supports service account IAM roles, I think less people will be using kiam. You can also increase the hop count on IMDSv2 to make NTH work without host networking, but we wanted to optimize for the super simple install that you get with NTH.  Since NTH isn't an API, it's just polling metadata, I think the risk is lower for the container to be compromised which makes host networking less of an issue.  

I'll also note that we're working on a new mode of NTH that will not rely on IMDS and instead use EventBridge + SQS. This new mode will not deploy as a daemonset and won't use host networking.

Reply:
@bwagner5 makes sense. ++